module.json

```diff
@@ -1,15 +1,15 @@
 {
     "id": "cfiles",
     "name": "Files",
-    "description": "Enhance your network with a file managment system",
+    "description": "Enhance your network with a file management system",
     "keywords": [
         "files",
         "docs",
-        "structre",
+        "structure",
         "organisation",
         "sharing"
     ],
-    "version": "0.13.2",
+    "version": "0.13.3",
     "humhub": {
         "minVersion": "1.8"
```

Comment:
Min Version, 1.10

Reply:
@luke- It was changed here - https://github.com/humhub/cfiles/pull/138/commits/8fcc3b1c11b3e49e0ab5c1e2a98fc5d5de73b8f1#diff-e6fdaf48ec339e2166f48043597d67d91464d12ef34fb958b12d6c93ba5f6fe3R14.